docs/_articles/en/lwc/testing.md

```diff
@@ -17,6 +17,22 @@ To jump to a test file, or a test case, click the test name in the sidebar.
 
 To clear your test results, click the refresh icon at the top of the sidebar (hover text: SFDX: Refresh Lightning Web Component Test Explorer).
 
+## Add Debug Configuration for Jest Tests
+
+Add a debug configuration to automatically update ```launch.json```.  As a prerequisite, you must have [node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/get-npm) installed.
+
+
+> **_NOTE:_** If you started from an empty project, first install [sfdx-lwc-jest](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.unit_testing_using_jest_installation) and then add the test configuration using the steps described.
+
+To quickly add Jest Test configuration to your standard project:
```

Comment:
Is it "a" Jest Test config? or only one, so "the" Jest Test config? Re-reading it, no article sounds strange to me.